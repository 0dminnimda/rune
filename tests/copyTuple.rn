t1 = (123, [1.25f32, 2.5f32], "test")
t2 = t1
println t1
println t2
t3 = (t1, t2, "foo")
t4 = t3
println t3
println t4
