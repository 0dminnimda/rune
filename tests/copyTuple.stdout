(123u64, "test")
